.gitignore

```diff
@@ -13,11 +13,13 @@
 
 # misc
 .DS_Store
+.env
 .env.local
 .env.development.local
 .env.test.local
 .env.production.local
 .env
+.vscode/
```

Comment:
As a general piece of advice, this is your personal preference and has nothing to do with stumptown-renderer. I would advance that you add to your `~/.gitignore_global` now that you're on the vs code band wagon. 
```
▶ cat ~/.gitignore_global | rg vscode
.vscode/
```

Reply:
Good point, will do!